packages/breadboard/breadboard.schema.json

```diff
@@ -1,6 +1,6 @@
 {
   "$schema": "https://json-schema.org/draft/2020-12/schema",
-  "$id": "https://raw.githubusercontent.com/breadboard-ai/breadboard/main/packages/breadboard/breadboard.schema.json",
+  "$id": "https://raw.githubusercontent.com/breadboard-ai/breadboard/@google-labs/breadboard@0.8.0/packages/breadboard/breadboard.schema.json",
```

Comment:
I'm a bit skeptical we'll keep this up to date with the npm version unless we automate it. But also, this is unlikely to change as frequently as our version numbers.

So maybe we should have independent versioning for the json schema?

(I guess also, what is the practical implication of $id? what actually matters is what URL people put in their $schema, and to control that maybe we should use git tags?)